backend/src/firebase/user/user.controller.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Czemu to wyłączasz?

Reply:
bo w plikach nest mi pokazuje cały plik na czerwono i do każdej linijki kod jest taki błąd "Replace `␍` with `;`"

Reply:
On ci to pokazuje ze względu na to, że masz pewne zasady ustalone w prettierze i np. linia jest za długa albo nie ma spacji gdzie powinny być. Spróbuj dodać prettiera dla frontu też, bo widzę, że chyba jest na BE tylko i wtedy puścić prettiera żeby ci wszystko poprawił to nie bedziesz musiał mieć tego.